### ISSUE TITLE
Author roles in DESCRIPTION

### ISSUE THREAD

**Issue by edwardsmarc** [NCC-CNC/wheretowork#340]
Having updated the pkgdown site, we should review who the included authors are for wtw, and what roles are assigned to each person.

This is what it currently looks like: https://ncc-cnc.github.io/wheretowork/

R packages can have any number of authors, but one author has to be assigned as the 'maintainer' of the package. That is currently Jeff. We might want to consider switching that to an NCC staffer so any users looking to contact the wtw dev team get directed to Dan, myself and Richard.

@ricschuster has proposed that each person receive the role of **author**, and we do not use the **contributor** role.

We can use the footer to add a custom message. I'm proposing we use that to identify Jeff as the creator of the package.

Here's my suggestion which is open for discussion...

Authors:
* Jeff Hanson
* Richard Schuster
* Joeseph Bennett
* Jaimie Vincent
* Dan Wismer
* Marc Edwards

Maintainer: 
* Dan Wismer (dan.wismer@natureconservancy.ca)
or
* Richard Schuster (richard.schuster@natureconservancy.ca)
or
* Nature Conservancy Canada (dan.wismer@natureconservancy.ca)

Footer: 
* Developed by Nature Conservancy Canada and Carleton University. Written by Jeff Hanson.

**Comment by DanWismer** [comment]
Maintainer as NCC I like. Is it possible to put multiple emails? My guess is not.

**Comment by edwardsmarc** [comment]
> Maintainer as NCC I like. Is it possible to put multiple emails? My guess is not.

No, but we can put hyperlinks so one option would be to put NCC as the maintainer with a hyperlink to the issues page. Then each of us could also have our own emails attached to our names.

**Comment by DanWismer** [comment]
> > Maintainer as NCC I like. Is it possible to put multiple emails? My guess is not.
> 
> No, but we can put hyperlinks so one option would be to put NCC as the maintainer with a hyperlink to the issues page. Then each of us could also have our own emails attached to our names.

That's a perfect solution

**Comment by jeffreyhanson** [comment]
Yeah, I agree this definitely needs updating! Since both @DanWismer and @edwardsmarc are leading development now, I'd suggest having your names in the first two author positions?

**Comment by josephrbennett** [comment]
Looks great to me! I'd still keep Jeff in first author position for now, since he did bulk of original work. But I'm OK either way, if others feel differently.
